source/_lovelace/button.markdown

```diff
@@ -11,76 +11,112 @@ The Button card allows you to add buttons to perform tasks.
 Screenshot of the Button card.
 </p>
 
+## Visual Editor
+
+<p class='img'>
+<img src='https://i.imgur.com/HJybKBf.jpg' alt='Screenshot of the button card Visual Editor'>
+
+### Fields
+
+- **Entity**. Choose a Home Assistant entity from the dropdown menu.
+- **Name**. Enter a name for the button. The default is the entity name.
+- **Icon**. Choose an icon to overwrite the default entity picture or entity icon. 
+- **Show Name?** Toggle to show or hide the button name. 
+- **Show Icon?** Toggle to show or hide the button icon.
+- **Icon Height**. Enter a custom height for the icon. The height is automatically set if no height is entered. 
+- **Theme**. Choose a card theme from the dropdown menu.
+- **Tap Action**. Choose an action from the dropdown menu. This action is taken on card tap. See [action documentation](/lovelace/actions/#tap-action).
+- **Hold Action**. Choose an action from the dropdown menu. This action is taken on card tap and hold. See [action documentation](/lovelace/actions/#hold-action).
+
+## YAML
+
+### Required Variables
+
 ```yaml
-type: button
-entity: light.living_room
+type: button # string
 ```
 
-{% configuration %}
```

Comment:
We should not remove the configuration blocks. Instead, we should improve on their output.

Reply:
Ok no problem. 

So just to clarify; are we trying to change the configuration block output on all pages that contain variables, or only for the Cards documentation? 

Definitely happy to keep working on this, however I'm not super familiar with Jekyll/Liquid. Where would I go to be able to change how the configuration block is output?

Reply:
Configuration is generated by this plugin: <https://github.com/home-assistant/home-assistant.io/blob/current/plugins/configuration.rb>

The styling can be found in the `sass` folder.